packages/web-runtime/src/components/UserMenu.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="userId">
+  <nav v-if="userId" :aria-label="$gettext('This is a main navigation')">
```

Comment:
I think we should go for a more specific string here, e.g. "Account menu" so the screenreader user gets informed about what navigation he's facing (otherwise he'd hear sth among the lines of *navigation item described as main navigation* and perhaps feel lost?

Reply:
Sounds reasonable, I'm just a bit thrown off by the requirement `give aria-label that described this as main navigation`. Do you think this is fulfilled this way?

Reply:
That's definitely wrong in the confluence page. `Account menu` or `User menu` is fine. Github has `View profile and more`, Google has `Google-account: <username> (<email>)`.

Reply:
The main navigation is the one in the app switcher.

Reply:
Haha okay. Because of the task in confluence I thought a site can have multiple main navigations. Now that makes sense, will adjust it 👍